tests/unit/memoize_test.py

```diff
@@ -0,0 +1,24 @@
+from mock import patch, Mock
+
+from dusty.memoize import memoized, reset_memoize_cache
+from ..testcases import DustyTestCase
+
+class TestMemoize(DustyTestCase):
+    def setUp(self):
+        super(TestMemoize, self).setUp()
+        self.counter = 0
+        @memoized
+        def memoized_fn():
+            self.counter += 1
+        self.memoized_fn = memoized_fn
+
+    def test_memoize(self):
+        self.memoized_fn()
```

Comment:
Probably worth adding a test to make sure the function is actually returning what you think it should be